packages/cli/snapshots/integration/generators/relation.belongs-to.integration.snapshots.js

```diff
@@ -7,9 +7,7 @@
 
 'use strict';
 
-exports[
-  `lb4 relation checks generated source class repository  answers {"relationType":"belongsTo","sourceModel":"Order","destinationModel":"Customer"} generates Order repository file with different inputs 1`
-] = `
+exports[`lb4 relation checks generated source class repository  answers {"relationType":"belongsTo","sourceModel":"Order","destinationModel":"Customer"} generates Order repository file with different inputs 1`] = `
```

Comment:
nitpick: double space after `repository`?

Reply:
Sorry, I'll move this commit to a separate PR. The snapshot is generated. If we need to fix the prompt, we should fix `cli`.

Reply:
See https://github.com/strongloop/loopback-next/pull/4761